src/main/java/com/smartcar/sdk/AuthClient.java

```diff
@@ -574,11 +579,43 @@ public Auth exchangeRefreshToken(String refreshToken) throws SmartcarException {
    * @throws SmartcarException when the request is unsuccessful
    */
   public boolean isCompatible(String vin, String[] scope) throws SmartcarException {
```

Comment:
Should we add `@deprecated` to this?

Reply:
I think we don't need to if we are going to allow defaults for country. If we are going to encourage using the country code (even if it is `US`) then I think we should `@deprecated`. Let me know what you think!

Reply:
The API will eventually enforce passing country, but I'm not sure what we should do in the SDKs. @sankethkatta thoughts?

Reply:
I think in the SDKs, so far, we've been adding in a `US` default. I'd be okay with having it required in the API, but let the SDKs set defaults for `US`. So @allisonc07 instead of a `null`, we can just pass in `US`.